Documentation/Setup/Settings.rst

```diff
@@ -78,14 +78,48 @@ Find detailed information in :ref:`TYPO3 Explained, Site handling <t3coreapi:sit
 Site settings
 -------------
 
-..  versionadded:: 12.0
-    Starting with TYPO3 v12 site settings can be saved in a file called
-    :file:`config/sites/my-site/settings.yaml`.
-
-Settings that only affect one site can be made in file
-:file:`config/sites/my-site/settings.yaml`. If that is necessary we will refer
-to this file in the documentation and authors of third party extensions might
-also refer you to it. Find detailed information in
+..  versionchanged:: 13.3
+    Starting with TYPO3 v13.3 site settings can also be edited in the site
+    settings editor.
+
+You can edit site settings in the module :guilabel:`Site Management > Settings`.
+Changes are written to file :file:`config/sites/my-site/settings.yaml`. If you
```

Comment:
Should we also mention the legacy path "typo3conf/sites/my-site/settings.yaml" for better orientiation? Approx. 50% of the installations out there are legacy.

Reply:
We are in the getting started guide here. And if you installed TYPO3 following the installaition chapter you have a composer-based installation.